tests/src/test/java/org/jboss/windup/tests/bootstrap/migrate/DisableTattletaleParameterTest.java

```diff
@@ -0,0 +1,53 @@
+package org.jboss.windup.tests.bootstrap.migrate;
+
+import org.jboss.windup.bootstrap.Bootstrap;
+import org.jboss.windup.rules.apps.tattletale.TattletaleRuleProvider;
+import org.jboss.windup.tests.bootstrap.AbstractBootstrapTestWithRules;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class DisableTattletaleParameterTest extends AbstractBootstrapTestWithRules {
+    @Rule
+    public final TemporaryFolder tmp = new TemporaryFolder();
+
+    @Rule
+    public final TemporaryFolder tmpAddonDir = new TemporaryFolder();
+
+    @Test
+    public void disableTattletaleParameterWithEapTarget() {
```

Comment:
are we following any approach for the name of the tests ? 
https://dzone.com/articles/7-popular-unit-test-naming

Reply:
I think our approach looks like `4. Feature to be tested`